README.md

```diff
@@ -134,6 +134,79 @@ into their own mount. This could be achieved with a configuration like:
                 - secret
 
 
+### Use global `:hierarchy` - optional
+By default, only the list of mounts is traversed as described above.
+However, when having configured:
+
+    :vault:
+        :use_hierarchy: true
+
+the `:hierarchy` source paths from the hiera configuration are used on top of each mount.
+This makes the behavior of the vault backend the same as other backends.
+Additionally, this enables usage of the third parameter to the hiera functions in puppet,
+the so-called 'override' parameter.
+
+Example: In case we have the following hiera config:
+
+    :backends:
+        - vault
+        - yaml
+
+    :hierarchy:
+      - nodes/%{fqdn}
```

Comment:
Our hierarchy list is in fact at least twice as long as in the example. But it makes sense indeed to make it even smaller for the example.